engine.py

```diff
@@ -115,25 +124,9 @@ def pre_app_init(self):
             if max_year >= self.get_setting(
                 "compatibility_dialog_min_version", max_next_year
             ):
-                MaxPlus.Core.EvalMAXScript(
-                    'messagebox "Warning - ' + msg + '" title: "Shotgun Warning"'
+                QtGui.QMessageBox.warning(
+                    None, "Shotgun Warning", "Warning - {0}".format(msg)
                 )
-
-            # and log the warning
-            self.log_warning(msg)
-
-        elif not self._is_at_least_max_2016():
```

Comment:
This `elif` is pointless. I doubt it ever got executed. If max is less than 2016, then the import at the top fails and the engine can't even be started.